2023-07-02 00:44:13,446	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 00:44:38,620	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 00:44:44,336	[WARNING]	(http.py:188)request	请求超时，请求连接: https://api.sgroup.qq.com/users/@me
2023-07-02 00:44:50,333	[WARNING]	(http.py:188)request	请求超时，请求连接: https://api.sgroup.qq.com/gateway/bot
2023-07-02 00:45:16,663	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 00:45:17,777	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 00:45:17,777	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 00:45:17,778	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 00:45:17,789	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 00:45:18,186	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 00:45:18,361	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 00:45:18,361	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 00:45:18,362	[INFO]	(测试.py:17)on_ready	robot 「OWER-测试中」 on_ready!
2023-07-02 00:45:32,401	[INFO]	(测试.py:20)on_at_message_create	https://qqchannel-profile-1251316161.file.myqcloud.com/16605616176d9cc4db9e8eebcd?t=1660561617
2023-07-02 00:45:32,401	[INFO]	(测试.py:23)on_at_message_create	白瞎子
2023-07-02 00:46:30,655	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 00:46:31,620	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 00:46:31,625	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 00:46:31,626	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 00:46:31,626	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 00:46:32,072	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 00:46:32,213	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「Ower-测试中」启动成功！
2023-07-02 00:46:32,214	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 00:47:10,722	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 00:47:11,664	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 00:47:11,664	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 00:47:11,665	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 00:47:11,665	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 00:47:12,105	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 00:47:12,274	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 00:47:12,274	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 00:47:19,319	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/13346486689812486705/members/17933814376511746278, 错误代码: 403, 返回内容: {'code': 50001, 'message': 'no privilege remove member'}, trace_id:8dc8327e908c5318598b3eec2140d41c
2023-07-02 00:49:04,882	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 00:49:05,954	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 00:49:05,955	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 00:49:05,955	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 00:49:05,955	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 00:49:06,367	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 00:49:06,544	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 00:49:06,544	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 00:49:16,539	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/13346486689812486705/members/17933814376511746278, 错误代码: 403, 返回内容: {'code': 50001, 'message': 'no privilege remove member'}, trace_id:34b443aa6c6c9dec859c8ca220620d62
2023-07-02 00:51:26,038	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 00:51:27,079	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 00:51:27,080	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 00:51:27,080	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 00:51:27,080	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 00:51:27,503	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 00:51:27,706	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「Ower-测试中」启动成功！
2023-07-02 00:51:27,707	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 00:51:43,720	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/12536101201533055903/members/8819902143901774960, 错误代码: 401, 返回内容: {'message': 'check guild auth not pass', 'code': 11264}, trace_id:76325a68a9d6dc13c16decf5b31dc9b9
2023-07-02 00:55:28,630	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 00:55:29,681	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 00:55:29,681	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 00:55:29,682	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 00:55:29,682	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 00:55:30,108	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 00:55:30,277	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「Ower-测试中」启动成功！
2023-07-02 00:55:30,277	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 00:55:30,278	[INFO]	(main.py:17)on_ready	robot 「Ower-测试中」 on_ready!
2023-07-02 00:58:22,175	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 00:58:23,257	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 00:58:23,257	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 00:58:23,257	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 00:58:23,257	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 00:58:23,600	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 00:58:23,750	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 00:58:23,750	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 00:58:23,751	[INFO]	(main.py:17)on_ready	robot 「OWER-测试中」 on_ready!
2023-07-02 01:00:09,873	[INFO]	(main.py:21)on_at_message_create	https://qqchannel-profile-1251316161.file.myqcloud.com/16605616176d9cc4db9e8eebcd?t=1660561617
2023-07-02 01:00:09,873	[INFO]	(main.py:24)on_at_message_create	白瞎子
2023-07-02 01:00:37,007	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:00:38,115	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:00:38,115	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:00:38,116	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:00:38,116	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:00:38,529	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:00:38,732	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:00:38,732	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:00:38,733	[INFO]	(main.py:17)on_ready	robot 「OWER-测试中」 on_ready!
2023-07-02 01:00:47,813	[INFO]	(main.py:21)on_at_message_create	https://qqchannel-profile-1251316161.file.myqcloud.com/16605616176d9cc4db9e8eebcd?t=1660561617
2023-07-02 01:00:47,813	[INFO]	(main.py:24)on_at_message_create	白瞎子
2023-07-02 01:02:15,742	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:02:16,815	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:02:16,815	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:02:16,815	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:02:16,816	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:02:17,204	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:02:17,353	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:02:17,354	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:03:55,642	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:03:56,644	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:03:56,644	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:03:56,644	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:03:56,645	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:03:56,972	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:03:57,123	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:03:57,124	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:05:22,964	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:05:24,211	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:05:24,211	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:05:24,212	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:05:24,212	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:05:24,669	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:05:24,820	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:05:24,821	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:05:36,440	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/13346486689812486705/messages, 错误代码: 500, 返回内容: {'code': 304033, 'message': 'internal error: direct message error'}, trace_id:38a4c3b5ff439e1e51b17d23be6b2c0b
2023-07-02 01:06:44,018	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:06:45,039	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:06:45,039	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:06:45,039	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:06:45,040	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:06:45,532	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:06:45,701	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:06:45,702	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:06:51,376	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/13346486689812486705/messages, 错误代码: 500, 返回内容: {'code': 304033, 'message': 'internal error: direct message error'}, trace_id:0aa1dddc607e3644b9b525d6d3f7a8be
2023-07-02 01:18:37,892	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:18:38,959	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:18:38,960	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:18:38,960	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:18:38,960	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:18:39,332	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:18:39,519	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:18:39,519	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:21:29,289	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:21:30,302	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:21:30,302	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:21:30,302	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:21:30,302	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:21:30,677	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:21:30,850	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:21:30,851	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:21:40,384	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/channels/560174013/messages, 错误代码: 500, 返回内容: {'code': 304022, 'message': 'push time out of range'}, trace_id:5c24b11c9dcfb12c7b555512e30a18e7
2023-07-02 01:22:08,749	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:22:09,799	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:22:09,799	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:22:09,799	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:22:09,799	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:22:10,240	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:22:10,373	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:22:10,374	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:27:10,899	[INFO]	(gateway.py:54)on_closed	[botpy] 关闭, 返回码: WSCloseCode.ABNORMAL_CLOSURE, 返回信息: None
2023-07-02 01:27:15,912	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:27:15,912	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:27:15,913	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:27:16,408	[INFO]	(gateway.py:169)ws_resume	[botpy] 重连启动...
2023-07-02 01:27:16,553	[INFO]	(gateway.py:85)on_message	[botpy] 机器人重连成功! 
2023-07-02 01:27:16,554	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:29:21,270	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/users/@me/dms, 错误代码: 500, 返回内容: {'code': 305004, 'message': 'recipient is not source guild member'}, trace_id:43e8b5b693281fc621d36bd703f2a6ba
2023-07-02 01:30:06,948	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:30:07,932	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:30:07,932	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:30:07,932	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:30:07,932	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:30:08,327	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:30:08,487	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:30:08,487	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:30:20,740	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/users/@me/dms, 错误代码: 500, 返回内容: {'code': 305004, 'message': 'recipient is not source guild member'}, trace_id:1e866fea574673e228b8c34bc947cff5
2023-07-02 01:35:55,492	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:35:56,507	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:35:56,508	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:35:56,508	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:35:56,508	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:35:56,924	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:35:57,076	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:35:57,077	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:36:43,769	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:36:44,732	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:36:44,733	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:36:44,733	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:36:44,733	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:36:45,097	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:36:45,292	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:36:45,292	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:37:16,098	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/users/@me/dms, 错误代码: 500, 返回内容: {'code': 305004, 'message': 'recipient is not source guild member'}, trace_id:e3c33e780a2c57d64813777f6432dd93
2023-07-02 01:38:09,618	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:38:10,579	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:38:10,579	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:38:10,580	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:38:10,580	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:38:10,987	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:38:11,128	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:38:11,128	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:38:29,510	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:38:30,495	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:38:30,495	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:38:30,495	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:38:30,495	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:38:30,892	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:38:31,020	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:38:31,021	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:39:24,245	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:39:25,347	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:39:25,347	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:39:25,347	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:39:25,347	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:39:25,716	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:39:25,844	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:39:25,844	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:39:51,203	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:39:52,139	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:39:52,140	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:39:52,140	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:39:52,140	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:39:52,541	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:39:52,688	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:39:52,688	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:41:07,817	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:41:08,712	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:41:08,712	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:41:08,712	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:41:08,712	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:41:09,071	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:41:09,217	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:41:09,217	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:41:20,753	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/13346486689812486705/messages, 错误代码: 500, 返回内容: {'code': 304033, 'message': 'internal error: direct message error'}, trace_id:b67cb64f5a3b30beecc1c725ed8d4115
2023-07-02 01:41:41,572	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:41:42,489	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:41:42,489	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:41:42,489	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:41:42,489	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:41:42,977	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:41:43,159	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:41:43,160	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:41:48,647	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/13346486689812486705/messages, 错误代码: 500, 返回内容: {'code': 304033, 'message': 'internal error: direct message error'}, trace_id:da5d2ec547b57e9babb82d555bdbf29b
2023-07-02 01:46:26,756	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:46:27,763	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:46:27,763	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:46:27,764	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:46:27,764	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:46:28,133	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:46:28,290	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:46:28,290	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:46:41,960	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/13346486689812486705/messages, 错误代码: 500, 返回内容: {'code': 304033, 'message': 'internal error: direct message error'}, trace_id:cb14590a1c70d983d7dcff8075c15ee3
2023-07-02 01:48:09,133	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:48:10,224	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:48:10,225	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:48:10,225	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:48:10,225	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:48:10,608	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:48:10,744	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:48:10,745	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:48:31,579	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:48:32,590	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:48:32,590	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:48:32,590	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:48:32,591	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:48:32,986	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:48:33,136	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:48:33,136	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:48:43,761	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/13346486689812486705/messages, 错误代码: 500, 返回内容: {'code': 304033, 'message': 'internal error: direct message error'}, trace_id:f40363929ba2228715f38e2873b4394b
2023-07-02 01:49:38,667	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:49:39,712	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:49:39,713	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:49:39,713	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:49:39,713	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:49:40,079	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:49:40,244	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:49:40,244	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:49:47,821	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/13346486689812486705/messages, 错误代码: 500, 返回内容: {'code': 304033, 'message': 'internal error: direct message error'}, trace_id:a514b5b24b16e7570516c28071f07b54
2023-07-02 01:53:12,179	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:53:13,089	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:53:13,089	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:53:13,090	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:53:13,090	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:53:13,421	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:53:13,519	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:53:13,519	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:53:23,124	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/13346486689812486705/messages, 错误代码: 500, 返回内容: {'code': 304033, 'message': 'internal error: direct message error'}, trace_id:0ac154dd072ffd75201841aefa8bb2d2
2023-07-02 01:54:35,961	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:54:36,664	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:54:36,665	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:54:36,665	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:54:36,665	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:54:36,977	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:54:37,158	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:54:37,158	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:54:50,525	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/1505776315428655041/messages, 错误代码: 500, 返回内容: {'code': 304022, 'message': 'push time out of range'}, trace_id:54936fb971f7750d3a7ed482a03ad1cf
2023-07-02 01:56:07,498	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:56:08,263	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:56:08,263	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:56:08,264	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:56:08,264	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:56:08,562	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:56:08,678	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:56:08,679	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:56:15,918	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/4116989596879722469/messages, 错误代码: 500, 返回内容: {'code': 304022, 'message': 'push time out of range'}, trace_id:7359925a724a9c6657bdceda539b9971
2023-07-02 01:58:06,617	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 01:58:07,334	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 01:58:07,334	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 01:58:07,335	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 01:58:07,335	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 01:58:07,625	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 01:58:07,742	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 01:58:07,742	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 01:58:22,093	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/16787801098731547314/messages, 错误代码: 500, 返回内容: {'code': 304022, 'message': 'push time out of range'}, trace_id:328819d87b1e8aab382e93c5afe456e8
2023-07-02 01:58:22,571	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/13346486689812486705/members/9282202926517215240, 错误代码: 403, 返回内容: {'code': 50001, 'message': 'no privilege remove member'}, trace_id:23d82ed0d536113d7d4236913e34f9cb
2023-07-02 01:59:02,235	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/1505776315428655041/messages, 错误代码: 500, 返回内容: {'code': 304022, 'message': 'push time out of range'}, trace_id:f6808db4c7aadb5c948caa5a93060e94
2023-07-02 02:00:44,689	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:00:45,691	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:00:46,530	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:00:46,530	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:00:46,531	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:00:46,531	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:00:46,879	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:00:46,982	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:00:46,983	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:01:17,237	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/16787801098731547314/messages, 错误代码: 500, 返回内容: {'code': 304022, 'message': 'push time out of range'}, trace_id:4ab8bd65da94b7bc251b17db29ae2014
2023-07-02 02:04:27,896	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:04:29,116	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:04:29,925	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:04:29,925	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:04:29,925	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:04:29,925	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:04:30,253	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:04:30,469	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:04:30,469	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:04:50,641	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/16787801098731547314/messages, 错误代码: 500, 返回内容: {'code': 304049, 'message': 'push dm message reach limit'}, trace_id:406c0e1057466b2e2c05e32367bff941
2023-07-02 02:05:11,283	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:05:12,166	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:05:12,166	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:05:12,166	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:05:12,166	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:05:12,494	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:05:12,707	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:05:12,707	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:05:20,466	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/4116989596879722469/messages, 错误代码: 500, 返回内容: {'code': 304022, 'message': 'push time out of range'}, trace_id:bc6205d8cb715e51e3c88a4eb7137b10
2023-07-02 02:05:39,735	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:05:40,430	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:05:40,430	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:05:40,430	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:05:40,430	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:05:40,756	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:05:40,853	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:05:40,853	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:05:47,449	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/16787801098731547314/messages, 错误代码: 500, 返回内容: {'code': 304049, 'message': 'push dm message reach limit'}, trace_id:51e23f95064af881bf21b91347a305fb
2023-07-02 02:06:24,020	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:06:24,803	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:06:24,803	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:06:24,803	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:06:24,803	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:06:25,089	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:06:25,214	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:06:25,214	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:06:38,750	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/16787801098731547314/messages, 错误代码: 500, 返回内容: {'code': 304049, 'message': 'push dm message reach limit'}, trace_id:be00be41a269980ab8c04ac4d3216e10
2023-07-02 02:06:39,405	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/560174013/mute, 错误代码: 401, 返回内容: {'message': 'check guild auth error', 'code': 11263}, trace_id:9afb2d526d3d30bbcc843b4a78e753ab
2023-07-02 02:07:28,448	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:07:29,326	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:07:29,326	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:07:29,326	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:07:29,326	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:07:29,640	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:07:29,796	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:07:29,796	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:07:54,665	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/16787801098731547314/messages, 错误代码: 500, 返回内容: {'code': 304049, 'message': 'push dm message reach limit'}, trace_id:694b083ecd83930e3a69d0e25788478f
2023-07-02 02:07:55,185	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/guilds/560174013/mute, 错误代码: 401, 返回内容: {'message': 'check guild auth error', 'code': 11263}, trace_id:cc108dacc8c7b706743691110674f293
2023-07-02 02:08:12,416	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:08:13,112	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:08:13,113	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:08:13,113	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:08:13,113	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:08:13,420	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:08:13,546	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:08:13,546	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:08:29,055	[ERROR]	(http.py:73)_handle_response	[botpy] 接口请求异常，请求连接: https://api.sgroup.qq.com/dms/16787801098731547314/messages, 错误代码: 500, 返回内容: {'code': 304049, 'message': 'push dm message reach limit'}, trace_id:f2736804fbff7f74375827fd16831627
2023-07-02 02:09:53,202	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:09:54,189	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:09:54,189	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:09:54,189	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:09:54,190	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:09:54,510	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:09:54,615	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:09:54,615	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:10:11,891	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:10:12,677	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:10:12,677	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:10:12,677	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:10:12,677	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:10:12,984	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:10:13,153	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:10:13,153	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:15:47,728	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:15:48,575	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:15:48,575	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:15:48,576	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:15:48,576	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:15:48,894	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:15:49,018	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:15:49,018	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:15:49,019	[INFO]	(main.py:17)on_ready	robot 「OWER-测试中」 on_ready!
2023-07-02 02:15:55,974	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:15:56,824	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:15:56,825	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:15:56,825	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:15:56,825	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:15:57,093	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:15:57,191	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:15:57,192	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:15:57,192	[INFO]	(main.py:17)on_ready	robot 「OWER-测试中」 on_ready!
2023-07-02 02:16:03,277	[INFO]	(main.py:21)on_at_message_create	http://thirdqq.qlogo.cn/g?b=oidb&k=SZO8VIpuDNNvDpt8y43hzA&kti=ZJ1iMQAAAAA&s=0&t=1676469450
2023-07-02 02:16:03,277	[INFO]	(main.py:24)on_at_message_create	氧化成硫
2023-07-02 02:16:50,482	[INFO]	(main.py:21)on_at_message_create	http://thirdqq.qlogo.cn/g?b=oidb&k=SZO8VIpuDNNvDpt8y43hzA&kti=ZJ1iMQAAAAA&s=0&t=1676469450
2023-07-02 02:16:50,483	[INFO]	(main.py:24)on_at_message_create	氧化成硫
2023-07-02 02:17:04,718	[INFO]	(main.py:21)on_at_message_create	http://thirdqq.qlogo.cn/g?b=oidb&k=SZO8VIpuDNNvDpt8y43hzA&kti=ZJ1iMQAAAAA&s=0&t=1676469450
2023-07-02 02:17:04,718	[INFO]	(main.py:24)on_at_message_create	氧化成硫
2023-07-02 02:17:41,786	[INFO]	(main.py:21)on_at_message_create	http://thirdqq.qlogo.cn/g?b=oidb&k=SZO8VIpuDNNvDpt8y43hzA&kti=ZJ1iMQAAAAA&s=0&t=1676469450
2023-07-02 02:17:41,786	[INFO]	(main.py:24)on_at_message_create	氧化成硫
2023-07-02 02:18:21,942	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:18:22,877	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:18:22,877	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:18:22,878	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:18:22,878	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:18:23,155	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:18:23,317	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:18:23,317	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:18:23,317	[INFO]	(main.py:17)on_ready	robot 「OWER-测试中」 on_ready!
2023-07-02 02:18:39,440	[INFO]	(main.py:21)on_at_message_create	http://thirdqq.qlogo.cn/g?b=oidb&k=SZO8VIpuDNNvDpt8y43hzA&kti=ZJ1iMQAAAAA&s=0&t=1676469450
2023-07-02 02:18:39,441	[INFO]	(main.py:24)on_at_message_create	氧化成硫
2023-07-02 02:19:30,014	[INFO]	(client.py:159)_bot_login	[botpy] 登录机器人账号中...
2023-07-02 02:19:30,899	[INFO]	(client.py:178)_bot_init	[botpy] 程序启动...
2023-07-02 02:19:30,900	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 02:19:30,900	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 02:19:30,900	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 02:19:31,237	[INFO]	(gateway.py:136)ws_identify	[botpy] 鉴权中...
2023-07-02 02:19:31,452	[INFO]	(gateway.py:80)on_message	[botpy] 机器人「OWER-测试中」启动成功！
2023-07-02 02:19:31,452	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
2023-07-02 02:19:31,453	[INFO]	(main.py:17)on_ready	robot 「OWER-测试中」 on_ready!
2023-07-02 02:19:39,603	[INFO]	(main.py:21)on_at_message_create	http://thirdqq.qlogo.cn/g?b=oidb&k=SZO8VIpuDNNvDpt8y43hzA&kti=ZJ1iMQAAAAA&s=0&t=1676469450
2023-07-02 02:19:39,603	[INFO]	(main.py:24)on_at_message_create	氧化成硫
2023-07-02 09:55:21,256	[INFO]	(gateway.py:54)on_closed	[botpy] 关闭, 返回码: WSCloseCode.ABNORMAL_CLOSURE, 返回信息: None
2023-07-02 09:55:26,271	[INFO]	(connection.py:59)multi_run	[botpy] 最大并发连接数: 1, 启动会话数: 1
2023-07-02 09:55:26,271	[INFO]	(client.py:236)bot_connect	[botpy] 会话启动中...
2023-07-02 09:55:26,272	[INFO]	(gateway.py:110)ws_connect	[botpy] 启动中...
2023-07-02 09:55:26,429	[INFO]	(gateway.py:169)ws_resume	[botpy] 重连启动...
2023-07-02 09:55:26,626	[INFO]	(gateway.py:85)on_message	[botpy] 机器人重连成功! 
2023-07-02 09:55:26,627	[INFO]	(gateway.py:217)_send_heart	[botpy] 心跳维持启动...
